test/test-utils.js

```diff
@@ -13,6 +13,9 @@
 // limitations under the License.
 
 (function(exports, global) {
+
+  var chai = require('chai/chai.js');
```

Comment:
Actually, this file is used on the browser test environment too. We need an alternative approach.